app/src/main/java/org/dhis2/usescases/teiDashboard/dashboardfragments/teidata/TEIDataFragment.java

```diff
@@ -425,6 +425,7 @@ private void createEvent(EventCreationType eventCreationType, Integer scheduleIn
 
     @Override
     public void showCatComboDialog(String eventId, CategoryCombo categoryCombo, List<CategoryOptionCombo> categoryOptionCombos) {
+        if (categoryCombo.code().equals("default")) return;
         CategoryComboDialog dialog = new CategoryComboDialog(
                 getAbstracContext(),
                 categoryCombo,
```

Comment:
@mmmateos @Sharmyn28 @Balcan I think something like this is being done in datasets. is this okay? Or we might need to improve it?

Thanks